code/modules/paperwork/photography.dm

```diff
@@ -199,7 +215,13 @@
 		atoms.Add(the_turf);
 		// As well as anything that isn't invisible.
 		for(var/atom/A in the_turf)
-			if(A.invisibility) continue
+			if(A.invisibility )
+				if(see_ghosts && istype(A,/mob/dead/observer))
+					var/mob/dead/observer/O = A

```

Comment:
this adds the mob to the list of atoms, this list of atoms is used to gather images from.
Preventing it going in the list prevents it getting an image generated in the picture.